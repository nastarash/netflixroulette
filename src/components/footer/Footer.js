import React from 'react';

function Footer() {
  return (
    <footer>
      <h1>netflixroullete</h1>
    </footer>
  );
}
export default Footer;
