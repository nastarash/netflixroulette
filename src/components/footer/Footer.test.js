import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

it('render correctly', () => {
  const AppComponent = renderer.create(<Footer />).toJSON();
  expect(AppComponent).toMatchSnapshot();
});
