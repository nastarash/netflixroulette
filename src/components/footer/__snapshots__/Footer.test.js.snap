// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<footer>
  <h1>
    netflixroullete
  </h1>
</footer>
`;
